frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx

```diff
@@ -197,7 +203,8 @@ export const stopPipelineRun: KebabAction = (kind: K8sKind, pipelineRun: Pipelin
 };
 
 const addTrigger: KebabAction = (kind: K8sKind, pipeline: Pipeline) => ({
-  label: 'Add Trigger',
+  // t('pipelines-plugin~Add Trigger')
```

Comment:
Please remove all the commented statements in this file and also rebase the PR.

Reply:
I believe these are necessary for the locale parser to generate the strings

Reply:
These comments are needed.